src/redux/reducers/messages.js

```diff
@@ -6,7 +6,8 @@ const defaultState = {
   messagesByID: {},
   unreadMessageIDs: [],
   feedbackByID: {},
-  showLoading: false,
+  loadingHistory: false,
```

Comment:
Added this to use the new loaded while the history messages are fetched from the server.